src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -13,6 +13,7 @@ import mesosphere.marathon.api.v2.json.AppUpdate
 import mesosphere.marathon.api.v2.json.Formats._
 import mesosphere.marathon.api.{ AuthResource, MarathonMediaType, RestResource }
 import mesosphere.marathon.core.appinfo.{ AppInfo, AppInfoService, AppSelector, TaskCounts }
+import mesosphere.marathon.core.appinfo.AppInfo.Embed
```

Comment:
We are in AppsResource - it should be clear. 
But I added the long version with line breaks.